scale/docs/rest/v6/node.rst

```diff
@@ -0,0 +1,239 @@
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| id                 | Integer           | The unique identifier of the model. Can be passed to the details API call.     |
+|                    |                   | (See :ref:`Node Details <rest_node_details>`)                                  |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| hostname           | String            | The full domain-qualified hostname of the node.                                |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| pause_reason       | String            | The reason this node is paused if is_paused is true. This is a descriptive     |
+|                    |                   | field for presentation to the user.                                            |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| is_paused          | Boolean           | True if the node is paused and will not accept new jobs for execution.         |
+|                    |                   | Remaining tasks for a previously executing job will complete.                  |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| is_active          | Boolean           | True if the node is actively participating in the cluster.                     |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| deprecated         | ISO-8601 Datetime | When the node was removed (is_active == False) from the cluster.               |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| created            | ISO-8601 Datetime | When the associated database model was initially created.                      |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+| last_modified      | ISO-8601 Datetime | When the associated database model was last saved.                             |
++--------------------+-------------------+--------------------------------------------------------------------------------+
+
+.. _rest_v6_node_update:
+
+v6 Node Update
+----------------------
+
+**Example GET /v6/nodes/{id} API call**
+
+Request: PATCH http://.../v6/nodes/{id}/
+
+Response: 200 OK
+
+ .. code-block:: javascript  
+ 
+   { 
+       "id": 4, 
+       "hostname": "host.com", 
+       "is_paused": false, 
+       "is_active": true, 
+       "deprecated": null, 
+       "created": "2015-06-15T17:18:52.414Z", 
+       "last_modified": "2015-06-17T20:05:16.041Z", 
+       } 
+   } 
+
```

Comment:
Let's change the PATCH update to return 204 with no response body. I think this will be the standard way we should do PATCH requests.